### PR TITLE
06 get api articles article id comments

### PR DESCRIPTION
**Pull request by Meli-Nzekele**
Error handling test for 06-get-api-articles-article_id-comments from Line 180 in app.test.js

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -114,14 +117,45 @@ describe("app", () => {
...
+          });
+        });
+    });
+    it("200: GET - responds with an array of comments for the correct article, ordered by date descending", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeSorted("created_at");
```
Comment (pippaamy) [inline __tests__/app.test.js]: needs second argument of descending! :)

**Review by pippaamy — COMMENTED** [review]
Just one argument to add in test then happy for you to merge :)

File: __tests__/app.test.js (hunk 1 above)
Comment (Meli-Nzekele) [reply __tests__/app.test.js]: No problem, Thank you :)

**Comment by Meli-Nzekele** [comment]
Argument added to test :)

**Pull request merged**